utils/rpms/daos.spec

```diff
@@ -470,15 +462,17 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # For avocado tests
 %{daoshome}/.build_vars.json
 %{daoshome}/.build_vars.sh
-
-%files client-tests-openmpi
-%{_bindir}/crt_launch
 %{_bindir}/daos_perf
 %{_bindir}/daos_racer
 %{_bindir}/daos_test
 %{_bindir}/dfs_test
 %{_bindir}/jobtest
 %{_libdir}/libdts.so
+%{_libdir}/libdpar.so
+
+%files client-tests-openmpi
+%{_bindir}/crt_launch
+%{_libdir}/libdpar_mpi.so
```

Comment:
Same question about whether this is needed for the server tests.

Reply:
It is needed for tests that are run with more than one rank.  Any tests that does mpirun ./test is fine but if they pass a hostlist of -np > 1, it is needed to also preload this library.  I don't think we run any server tests this way in any of our testing.

Do we use RPMs in our testing by default or should I run something explicitly to test that?

Reply:
> It is needed for tests that are run with more than one rank. Any tests that does mpirun ./test is fine but if they pass a hostlist of -np > 1, it is needed to also preload this library.

So basically `libdpar.so` is to allow tests to be written with MPI, but can be used when they don't actually run with multiple nodes?  Do we do a lot of this?

> Do we use RPMs in our testing by default

In functional testing, yes.

> or should I run something explicitly to test that?

I must have lost track.  Test what exactly?

Reply:
> 

Yes, I would say most of the runs we do are on a single node, single rank and do not need MPI.  We only have a handful of tests that need libdpar_mpi.so to be run in CI.